analyzer/tests/functional/analyze/test_analyze.py

```diff
@@ -284,7 +284,6 @@ def test_failure(self):
         # We expect a failure archive to be in the failed directory.
         failed_files = os.listdir(failed_dir)
         self.assertEquals(len(failed_files), 1)
-        self.assertIn("failure.c", failed_files[0])
```

Comment:
Why was this assert removed? Shouldn't we check if the file is in the zip?

Reply:
This will be checked here:

https://github.com/Ericsson/codechecker/blob/68a1d54245e6eaba3c0ee6b7a27ca916bf0baa50/analyzer/tests/functional/analyze/test_analyze.py#L301-L303
where `source_file` is:
https://github.com/Ericsson/codechecker/blob/68a1d54245e6eaba3c0ee6b7a27ca916bf0baa50/analyzer/tests/functional/analyze/test_analyze.py#L257